README.md

```diff
@@ -57,7 +57,7 @@ ___
 ___
 
 ## Attributions
-Please see [`__Attributions.md`](https://github.com/Courseplay/Courseplay_FS22/blob/master/__Attributions.md) for more information.
+Please see [`__Attributions.md`](https://github.com/Courseplay/Courseplay_FS22/master/__Attributions.md) for more information.
```

Comment:
Github supports relative links in their Markdown files, currently you are attempting to point to an absolute path, along with that you are pointing to the `master` branch, which this repository is not using, they are using `main` which is the default for Github.
If you change this to
```md
Please see [`__Attributions.md`](__Attributions.md) for more information.
```
It should work just fine

Reply:
Thanks for the suggestion.
Run to apply the change and post a new commit.